.github/workflows/build.yml

```diff
@@ -49,8 +49,9 @@ jobs:
 
 
       - name: Upload Artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
+          name: vyper-${{ matrix.os }}
```

Comment:
why does this use `matrix.os` but the other uses `runner.os`?

Reply:
Done! I verified that `runner.os` works too.